### DISCUSSION TITLE
凭什么以骸只有挨打的份！

### DISCUSSION THREAD

**Discussion by ADAD33** [share121/inter-knot#2260] [其他]
可恶的绳匠竟然天天来空洞打我们，昨天我隔壁邻居的二大爷的舅舅的侄子的孙女的七大姑的孩子就被一个拿着大剪刀的女仆打死了！！！
我一定要锻炼成为最强的以骸，让这些绳匠付出代价！！！

**Comment by whatcanisaylaoda** [comment]
远远远远房亲戚？

**Comment by pomegrap** [comment]
是啊，那天我正在空洞里面散步，他们喊着什么“石头剪子布！” ”苍角来咯！“ ”鲨旋风！“，就把我朋友打死了，我准备搬到芭莱大厦那边去了，那边据说有两个很厉害的以骸，什么绳匠什么盗洞客库库乱揍！你是不知道，他们会召唤雷电！！死路屠夫听过吧，知道为什么最近没消息了吗，被他们劈死了！我当时就在附近！听我一句劝，快搬走吧。

**Reply by ADAD33** [reply]
谢谢兄弟的推荐

**Reply by Kamisato100** [reply]
要不你还是搬去防卫战那边吧，那边的以骸贼猛打代理人一拳一个小朋友不吱声的

**Comment by Genshin-Neuvillette** [comment]
最近看到了以骸的本子，是你吗？<img src="https://img-static.mihoyo.com/communityweb/upload/d9797181a9d6f097522b26e393964592.png" alt="若陀龙王-迷茫" width="75" >
